### DISCUSSION TITLE
Does Anti-Capitalism hide blazed posts?

### DISCUSSION THREAD

**Discussion by inklesspen** [AprilSylph/XKit-Rewritten#1091] [Help Desk] [answered]
And if it does, can an option be added to **not** hide blazed posts?

**Comment by marcustyphoon** [comment, marked as answer]
It currently does, yes. (We didn't do anything specific in order to make this occur, if I remember correctly; when blazed posts were added, the code hid them, so that was that.)

Maybe! (Of course, an ad-free subscription is one way to do this.)

**Reply by inklesspen** [reply]
I do currently have an ad-free subscription, but unfortunately the ad-free subscription does not allow me to opt out of things like the recent "Cruel Summer" dashboard takeover, with the animated "Turn your computer off" sticker. I found that to be very frustrating, and would like to not see it.

**Reply by inklesspen** [reply]
But of course I still want to see blazed posts.
